program/rust/src/tests/test_upd_price_v2.rs

```diff
@@ -77,12 +77,11 @@ fn test_upd_price_v2() -> Result<(), Box<dyn std::error::Error>> {
         assert_eq!(price_data.comp_[0].latest_.status_, PC_STATUS_TRADING);
         assert_eq!(price_data.valid_slot_, 0);
         assert_eq!(price_data.agg_.pub_slot_, 1);
-        assert_eq!(price_data.agg_.price_, 0);
```

Comment:
This file is a close of test_upd_price.rs that was meant to be run only for pythnet.
Since we don't have the pythnet feature anymore I think we should merge `test_upd_price.rs` and `test_upd_price_v2.rs`